Godeps/_workspace/src/github.com/Clever/gearadmin/Makefile

```diff
@@ -0,0 +1,20 @@
+SHELL := /bin/bash
+PKG = github.com/Clever/gearadmin
+SUBPKGS =
```

Comment:
The convention when using Godep (and vendoring in general) is to commit the
code of your dependencies

This is what we've done in other godep'd repos


On Jan 5, 2015, at 9:09 PM, Alex Zylman <notifications@github.com> wrote:

In Godeps/_workspace/src/github.com/Clever/gearadmin/Makefile
<https://github.com/Clever/gearman-load-logger/pull/1#discussion-diff-22509096>
:

> @@ -0,0 +1,20 @@
> +SHELL := /bin/bash
> +PKG = github.com/Clever/gearadmin
> +SUBPKGS =

Yeah, can we gitignore this?

—
Reply to this email directly or view it on GitHub
<https://github.com/Clever/gearman-load-logger/pull/1/files#r22509096>.